target_api/settings.py

```diff
@@ -41,9 +41,20 @@
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
+    'django.contrib.sites',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
```

Comment:
Maybe add a comment in order to document better the installed apps or splits in different arrays and joined together